ui/components/Logout.tsx

```diff
@@ -1,17 +1,22 @@
 'use client';
 import { Button } from '@/lib/Button';
+import { useSession } from 'next-auth/react';
+import { useRouter } from 'next/navigation';
 
 export default function Logout() {
-  return (
-    <Button
-      style={{ backgroundColor: 'white', border: '1px solid rgba(0,0,0,0.15)' }}
-      onClick={() =>
-        fetch('/api/logout', { method: 'POST' }).then((res) =>
-          location.assign('/login')
-        )
-      }
-    >
-      Log out
-    </Button>
-  );
+  const { data: session } = useSession();
+  const router = useRouter();
+  if (session) {
```

Comment:
should there be an else branch?

Reply:
We don't want to show the logout button in case we don't detect a session (is equivalent to not having auth enabled)